### ISSUE TITLE
Example never terminates in consensus

### ISSUE THREAD

**Issue by wildbunny** [ripple/simulator#4]
I built in VS2013, debug or release. Each time the example never terminates in consensus - it just gets stuck in a loop with no messages to process.

**Comment by vinniefalco** [comment]
For real?

**Comment by wildbunny** [comment]
Yes, for real. I didn't have time to figure out why, but it just gets completely stuck with no messages to process inside the loop commented '// run simulation', but we do _not_ get the 'Fatal: Radio Silence' message.
